src/tactic/ui/app/page_header_wdg.py

```diff
@@ -363,9 +363,16 @@ def get_display(my):
         if not WebContainer.get_web().is_IE():
             widget.add_style("float: right")
 
-        from tactic.ui.widget import SingleButtonWdg
-        #button = SingleButtonWdg(title='Open Project', icon=IconWdg.PROJECT, show_arrow=True)
-        button = SingleButtonWdg(title='Open Project', icon="BS_FOLDER_OPEN", show_arrow=True)
+        from tactic.ui.widget import SingleButtonWdg, IconButtonWdg
+
+        display = my.kwargs.get("display")
```

Comment:
This was a bit of a hack.  I have made the ProjectSelectWdg accept "icon" as a kwarg instead, so that the icon can be overridden to whatever you wish.